.github/workflows/update-render-staging-service.yml

```diff
@@ -0,0 +1,34 @@
+name: Update render staging service
+
+on:
+  create:
```

Comment:
Use push ?

Since we want to redeploy whenever there is cherrypick on the release branch.

Reply:
Render seems to support this. 
<img width="978" alt="CleanShot 2022-07-02 at 22 45 20@2x" src="https://user-images.githubusercontent.com/87714218/177005408-fd84b547-9bca-4932-b04b-4855e6409721.png">

Reply:
Oh, so whenever we start a new release branch, we still need to visit render to manually change the branch name. And we couldn't remove the manual step?

Reply:
This GitHub action can change the branch name on render when we start a new release branch. Then, if we cherry-pick some commits to the release branch, `Auto-Deploy` can detect it, and re-deploy the web service.

Reply:
I see. Then I still prefer to use push here since it's more explicit and specified in the code. The auto-deploy feature is not obvious unless the reader knows that.

Reply:
SGTM.

Reply:
Done. PTAL.

Reply:
push already covers create (our build-push-release-image uses this convention)